.github/workflows/ci.yml

```diff
@@ -128,3 +128,25 @@ jobs:
       - name: Show all images built
         run: docker images
   
+  run-postman-collection:
+    needs: build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Pull Images From DockerHub and Start crAPI
+        run: docker-compose -f deploy/docker/docker-compose.yml --compatibility up -d
```

Comment:
We need to prefix the docker-compose with VERSION variable so it always uses the currently built images and not remote images. 
Also combine this to the previous JOB to use leverage the QEMU setup.
`VERSION=${{ env.TAG_NAME }} docker-compose -f deploy/docker/docker-compose.yml --compatibility up -d`

Reply:
I have made the changes and pushed to my branch.